src/data/services/EnterpriseDataApiService.js

```diff
@@ -5,35 +5,39 @@ import { getAccessToken } from '../../utils';
 
 class EnterpriseDataApiService {
   // TODO: This should access the data-api through the gateway instead of direct
-  static enterpiseBaseUrl = `${config.DATA_API_BASE_URL}/enterprise/api/v0/enterprise/`;
+  static enterpriseBaseUrl = `${config.DATA_API_BASE_URL}/enterprise/api/v0/enterprise/`;
 
   static fetchCourseEnrollments(enterpriseId, options) {
     const queryParams = {
       page: 1,
       page_size: 50,
       ...options,
     };
-    let enrollmentsUrl = `${this.enterpiseBaseUrl}${enterpriseId}/enrollments/`;
+    const enrollmentsUrl = `${this.enterpriseBaseUrl}${enterpriseId}/enrollments?${qs.stringify(queryParams)}`;
```

Comment:
Changed this to always pass in the paging options, even on initial load. Otherwise we rely on the backend to send us the correct `page_size` but the backend is currently configured to only send 10. This caused an error when we tried to page to the second page if we had > 10 but < 50.